proj/src/main/webapp/static/images/friendsChatting.svg

```diff
@@ -0,0 +1 @@
10.902344 3.210937-2.804687 8.09375-2.476562 10.902343.734376 1.464844 1.679687 3.695313 2.640624 6.117188 2.640624s4.65625-.960937 6.121093-2.640624c2.808594-3.210938 7.691407-3.539063 10.902344-.730469 3.214844 2.808593 3.542969 7.6875.734375 10.902343-4.402343 5.035157-10.875 7.921876-17.757812 7.921876zm0 0" fill="#3a484a"/><path d="m.589844 379.375.011718.011719c67.195313-7.433594 125.097657-72.335938 125.097657-72.335938s57.90625 64.902344 125.101562 72.335938c0 0 0 0 .007813-.011719-2.542969-68.226562-57.578125-122.734375-125.109375-122.734375-67.527344 0-122.5625 54.507813-125.109375 122.734375zm0 0" fill="#3a484a"/><path d="m136.519531 257.121094c-3.566406-.3125-7.171875-.480469-10.820312-.480469-67.527344 0-122.5625 54.507813-125.109375 122.738281l.011718.007813c7.425782-.820313 13.566407-2.019531 20.699219-4.078125 4.433594-62.507813 54.144531-112.859375 115.21875-118.1875zm0 0" fill="#323e40"/><path d="m203.238281 174.039062v-77.175781c0-27.570312-22.347656-49.921875-49.921875-49.921875h-102.894531c-27.570313 0-49.921875 22.351563-49.921875 49.921875v70.527344c0 27.570313 22.351562 49.917969 49.921875 49.917969h176.726563zm0 0" fill="#58b78c"/><g fill="#efedee"><path d="m155.757812 104.589844h-107.777343c-4.265625 0-7.726563-3.460938-7.726563-7.726563 0-4.269531 3.460938-7.726562 7.726563-7.726562h107.777343c4.269532 0 7.726563 3.457031 7.726563 7.726562 0 4.265625-3.457031 7.726563-7.726563 7.726563zm0 0"/><path d="m155.757812 139.851562h-107.777343c-4.265625 0-7.726563-3.457031-7.726563-7.726562 0-4.265625 3.460938-7.726562 7.726563-7.726562h107.777343c4.269532 0 7.726563 3.460937 7.726563 7.726562 0 4.269531-3.457031 7.726562-7.726563 7.726562zm0 0"/><path d="m115.195312 175.117188h-67.214843c-4.265625 0-7.726563-3.460938-7.726563-7.726563 0-4.269531 3.460938-7.726563 7.726563-7.726563h67.214843c4.265626 0 7.726563 3.457032 7.726563 7.726563 0 4.265625-3.460937 7.726563-7.726563 7.726563zm0 0"/></g><path d="m305.15625 426.234375v-77.179687c0-27.570313 22.351562-49.917969 49.921875-49.917969h102.894531c27.570313 0 49.921875 22.347656 49.921875 49.917969v70.527343c0 27.570313-22.351562 49.921875-49.921875 49.921875h-176.726562zm0 0" fill="#f2c42c"/><path d="m460.414062 356.785156h-107.777343c-4.269531 0-7.726563-3.460937-7.726563-7.726562 0-4.269532 3.457032-7.726563 7.726563-7.726563h107.777343c4.269532 0 7.726563 3.457031 7.726563 7.726563 0 4.265625-3.460937 7.726562-7.726563 7.726562zm0 0" fill="#efedee"/><path d="m460.414062 392.046875h-107.777343c-4.269531 0-7.726563-3.460937-7.726563-7.726563 0-4.265624 3.457032-7.726562 7.726563-7.726562h107.777343c4.269532 0 7.726563 3.460938 7.726563 7.726562 0 4.265626-3.460937 7.726563-7.726563 7.726563zm0 0" fill="#efedee"/><path d="m460.414062 427.3125h-67.214843c-4.265625 0-7.726563-3.460938-7.726563-7.726562 0-4.269532 3.460938-7.726563 7.726563-7.726563h67.214843c4.265626 0 7.726563 3.457031 7.726563 7.726563 0 4.265624-3.460937 7.726562-7.726563 7.726562zm0 0" fill="#efedee"/></svg>
```

Comment:
What's the source of this image?  Did you draw it?  If not, there needs to be attribution and you need to make sure that it's license-compatible before committing.

Reply:
Thank you for reminding me.  The website has free licensing with attribution.  I am going to add the tag in the footer.

Reply:
That looks good to me!